modules/candidate_list/test/candidate_listTest.php

```diff
@@ -352,7 +352,6 @@ function testPscidLink()
     {
         $this->safeGet($this->url . "/candidate_list/");
         $link = self::$pscidLink;
-        sleep(1);
```

Comment:
This was already doing a safeClick and using safeFindElement, so shouldn't be necessary.